src/main/java/edu/hm/hafner/analysis/parser/CrossCoreEmbeddedStudioParser.java

```diff
@@ -0,0 +1,60 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.LookaheadParser;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.util.LookaheadStream;
+
+/**
+ * A parser for the CrossCoreEmbeddedStudio (CCES) log files.
+ */
+public class CrossCoreEmbeddedStudioParser extends LookaheadParser {
+    private static final long serialVersionUID = 5490211629355204910L;
```

Comment:
`5490211629355204910L` was copied from `Gcc4CompilerParser.java`.

How to generate it otherwise?